src/utils/uri.ts

```diff
@@ -38,30 +35,32 @@ export class Erc1319URI {
           `Invalid package ID: ${packageId}. URI must include both a package name and version.`,
         );
       } else {
-        return packageId;
+        let [packageName, version] = packageId
+        let namespacedAsset = pathElements.splice(1).join("/")
```

Comment:
Worth having a dedicated function to derive the `namespacedAsset` (which can be tested for edge cases)?

Reply:
Hmmm, definitely down the road - but for the current context the `namespacedAsset` is a pretty superfluous datapoint